modules/hal_silabs/wiseconnect/nwp_fw_version.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2025 Silicon Laboratories Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* This value needs to be updated when the Wiseconnect SDK (hal_silabs/wiseconnect) is updated
+ * Actually mapped to Wiseconnect SDK 3.5.0
+ */
+#define SIWX91X_NWP_FW_EXPECTED_VERSION "B.2.14.5.0.0.10"
```

Comment:
Wouldn't it make more sense for this define to come from the HAL tree, i.e. the HAL tree "tells us" which version it is, and then we check at runtime that it matches. I think it might make sense to propose such a define to upstream wiseconnect.